cmd/identity/certificate_authority.go

```diff
@@ -211,5 +211,5 @@ func cmdCAExtensions(cmd *cobra.Command, args []string) (err error) {
 		return err
 	}
 
-	return printExtensions(ca.Cert.Raw, ca.Cert.ExtraExtensions)
+	return printExtensions(ca.Cert.Raw, ca.Cert.Extensions)
```

Comment:
(clarifying not suggesting otherwise lol) why are we using `Extensions` and not `ExtraExtensions` here?

Reply:
Basically, `Extensions` is used for reading extensions in a certificate; it's what's populated when deserializing a certificate. `ExtraExtensions` is used for writing extensions; it's not populated when parsing and is encoded into the certificate bytes when serializing (which is done with the `x509.CreateCertificate` function).

Reply:
see https://golang.org/pkg/crypto/x509/#Certificate
and https://golang.org/pkg/crypto/x509/#CreateCertificate